forge-gui/res/cardsfolder/upcoming/truth_or_consequences.txt

```diff
@@ -0,0 +1,7 @@
+Name:Truth or Consequences
+ManaCost:2 U R
+Types:Sorcery
+A:SP$ Vote | Defined$ Player | Secret$ True | VoteType$ Truth,Consequences | VoteTruth$ DBDraw | VoteConsequences$ DBDamage | EachVote$ True | SpellDescription$ Secret council — Each player secretly votes for truth or consequences, then those votes are revealed. You draw cards equal to the number of truth votes. Then choose an opponent at random. Truth or Consequences deals 3 damage to that player for each consequences vote.
```

Comment:
isn't EachVote wrong?

Reply:
@Northmoc Thoughts

Reply:
Hmm yeah EachVote doesn't seem right here... I think you should only draw once X cards (X=truth) then random opp should get dealt X/Thrice damage once (X=cons)

Matters for triggers I think